spark-plugins/src/main/java/co/cask/hydrator/plugin/batch/spark/NaiveBayesClassifier.java

```diff
@@ -83,11 +83,6 @@ public Config(String fileSetName, String path, String fieldToClassify, String fi
     }
   }
 
-  // for unit tests, otherwise config is injected by plugin framework.
-  public NaiveBayesClassifier(Config config) {
```

Comment:
yea, I just thought it was unnecessary (since we dont use it in test cases), so just removed it.